### DISCUSSION TITLE
Welcome to charty Discussions!

### DISCUSSION THREAD

**Discussion by hi-manshu** [hi-manshu/Charty#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alek43** [comment]
Can you give me some sample convert from json to group bar chart, i have json like this 
[
    {
        "salesname": "Yossy Riesiane",
        "kgs": 1875.0,
        "jmlcust": 2,
        "tontarget": 0
    },
    {
        "salesname": "Yan Abdi",
        "kgs": 64150.0,
        "jmlcust": 16,
        "tontarget": 0
    },
    {
        "salesname": "Kelvin Ferbrianto",
        "kgs": 3250.0,
        "jmlcust": 2,
        "tontarget": 0
    },
    {
        "salesname": "Liana",
        "kgs": 27650.0,
        "jmlcust": 7,
        "tontarget": 0
    }
]

i try to convert it by using this function
private fun convertData(listSalesOrder:List<SarmutMarketing>):List<GroupBarData>{
    val listConvert:MutableList<GroupBarData> = mutableListOf<GroupBarData>()
    for(salesOrder in listSalesOrder){
        val data= GroupBarData(label=salesOrder.salesname,
            dataPoints= listOf(salesOrder.jmlcust.toFloat(),salesOrder.kgs.toFloat(),salesOrder.tontarget.toFloat()),
            chartColors)
        listConvert.add(data)
    }
    return listConvert.toList()
}

But i got erorr

**Comment by hi-manshu** [comment]
What was the error?
